### PR TITLE
Removed jquery-ui from script registration as WP natively includes this library

### PR DESCRIPTION
**Pull request by GEOFAIRY**
Work item 54673 for details

### CODE REVIEW

File: admin/class-arlo-for-wordpress-admin.php [hunk 1]
```diff
@@ -195,17 +195,16 @@ public function enqueue_admin_scripts() {
...
 
 		$screen = get_current_screen();
 		if ( $this->plugin_screen_hook_suffix == $screen->id ) {
 			wp_enqueue_script( $this->plugin_slug . '-lsapiclient', plugins_url( 'assets/js/lib/ls-apiclient-1.2.0.min.js', __FILE__ ), array( 'jquery' ), VersionHandler::VERSION, true );
-			wp_enqueue_script( $this->plugin_slug . '-jquery-ui', plugins_url( 'assets/js/lib/jquery-ui.min.js', __FILE__ ), array( 'jquery' ), VersionHandler::VERSION, true );
 			wp_enqueue_script( $this->plugin_slug . '-codemirror', plugins_url( 'assets/js/lib/codemirror.js', __FILE__ ), array(), VersionHandler::VERSION, true );
 			wp_enqueue_script( $this->plugin_slug . '-codemirror-css', plugins_url( 'assets/js/lib/codemirror-css.js', __FILE__ ), array(), VersionHandler::VERSION, true );
 			wp_enqueue_script( $this->plugin_slug . '-arlo-for-wordpress-script', plugins_url( 'assets/js/arlo_for_wordpress.js?20170424', __FILE__ ), array( 'jquery' ), VersionHandler::VERSION, true );
-			wp_enqueue_script( $this->plugin_slug . '-admin-script', plugins_url( 'assets/js/admin.js?20170424', __FILE__ ), array( 'jquery' ), VersionHandler::VERSION, true );
```
Comment (ry167) [inline admin/class-arlo-for-wordpress-admin.php:208]: There should be a dependency code we can include in the third parameter to tell WP we need jQuery UI

**Review by ry167 — CHANGES_REQUESTED** [review]

File: admin/class-arlo-for-wordpress-admin.php [hunk 2]
```diff
@@ -203,10 +203,10 @@ public function enqueue_admin_scripts() {
...
-			wp_enqueue_script( $this->plugin_slug . '-fancybox', 'https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.0.47/jquery.fancybox.min.js', array('jquery'), '3.3.7', true );
-			wp_enqueue_script( $this->plugin_slug . '-plugin-script-cookie', plugins_url( '../public/assets/js/libs/js.cookie.js', __FILE__ ), array( 'jquery' ), VersionHandler::VERSION );
+			wp_enqueue_script( $this->plugin_slug . '-arlo-for-wordpress-script', plugins_url( 'assets/js/arlo_for_wordpress.js?20170424', __FILE__ ), array( 'jquery', 'jquery-ui-core' ), VersionHandler::VERSION, true );
+			wp_enqueue_script( $this->plugin_slug . '-admin-script', plugins_url( 'assets/js/admin.js?20170424', __FILE__ ), array( 'jquery', 'jquery-ui-core' ), VersionHandler::VERSION, true );
+			wp_enqueue_script( $this->plugin_slug . '-fancybox', 'https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.0.47/jquery.fancybox.min.js', array('jquery', 'jquery-ui-core'), '3.3.7', true );
+			wp_enqueue_script( $this->plugin_slug . '-plugin-script-cookie', plugins_url( '../public/assets/js/libs/js.cookie.js', __FILE__ ), array( 'jquery', 'jquery-ui-core' ), VersionHandler::VERSION );
```
Comment (ry167) [inline admin/class-arlo-for-wordpress-admin.php:209]: I don't think this will need it?

File: admin/class-arlo-for-wordpress-admin.php [hunk 3]
```diff
@@ -203,10 +203,10 @@ public function enqueue_admin_scripts() {
...
 			wp_enqueue_script( $this->plugin_slug . '-codemirror-css', plugins_url( 'assets/js/lib/codemirror-css.js', __FILE__ ), array(), VersionHandler::VERSION, true );
-			wp_enqueue_script( $this->plugin_slug . '-arlo-for-wordpress-script', plugins_url( 'assets/js/arlo_for_wordpress.js?20170424', __FILE__ ), array( 'jquery' ), VersionHandler::VERSION, true );
-			wp_enqueue_script( $this->plugin_slug . '-admin-script', plugins_url( 'assets/js/admin.js?20170424', __FILE__ ), array( 'jquery' ), VersionHandler::VERSION, true );
-			wp_enqueue_script( $this->plugin_slug . '-fancybox', 'https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.0.47/jquery.fancybox.min.js', array('jquery'), '3.3.7', true );
-			wp_enqueue_script( $this->plugin_slug . '-plugin-script-cookie', plugins_url( '../public/assets/js/libs/js.cookie.js', __FILE__ ), array( 'jquery' ), VersionHandler::VERSION );
+			wp_enqueue_script( $this->plugin_slug . '-arlo-for-wordpress-script', plugins_url( 'assets/js/arlo_for_wordpress.js?20170424', __FILE__ ), array( 'jquery', 'jquery-ui-core' ), VersionHandler::VERSION, true );
```
Comment (ry167) [inline admin/class-arlo-for-wordpress-admin.php:206]: does this have a dependency on the cookie script? May as well add it while we are tidying this up.

File: admin/class-arlo-for-wordpress-admin.php [hunk 4]
```diff
@@ -203,10 +203,10 @@ public function enqueue_admin_scripts() {
...
+			wp_enqueue_script( $this->plugin_slug . '-admin-script', plugins_url( 'assets/js/admin.js?20170424', __FILE__ ), array( 'jquery', 'jquery-ui-core' ), VersionHandler::VERSION, true );
+			wp_enqueue_script( $this->plugin_slug . '-fancybox', 'https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.0.47/jquery.fancybox.min.js', array('jquery', 'jquery-ui-core'), '3.3.7', true );
```
Comment (ry167) [inline admin/class-arlo-for-wordpress-admin.php:208]: This doesn't depend on jquery UI https://fancyapps.com/fancybox/3/docs/

**Review by ry167 — CHANGES_REQUESTED** [review]

File: admin/class-arlo-for-wordpress-admin.php [hunk 5]
```diff
@@ -195,19 +195,18 @@ public function enqueue_admin_scripts() {
...
 			return;
 		}
-		
+
 		wp_enqueue_script( $this->plugin_slug . '-admin-global-script', plugins_url( 'assets/js/admin_public.js?20170424', __FILE__ ), array( 'jquery' ), VersionHandler::VERSION, true );
 
 		$screen = get_current_screen();
 		if ( $this->plugin_screen_hook_suffix == $screen->id ) {
 			wp_enqueue_script( $this->plugin_slug . '-lsapiclient', plugins_url( 'assets/js/lib/ls-apiclient-1.2.0.min.js', __FILE__ ), array( 'jquery' ), VersionHandler::VERSION, true );
-			wp_enqueue_script( $this->plugin_slug . '-jquery-ui', plugins_url( 'assets/js/lib/jquery-ui.min.js', __FILE__ ), array( 'jquery' ), VersionHandler::VERSION, true );
```
Comment (mcdong429) [inline admin/class-arlo-for-wordpress-admin.php:204]: Can you also remove the jquery-ui.min.js and jquery-ui.min.css files if they are no longer needed? 

As well as the reference to jquery-ui.min.css at [#L176](https://github.com/ArloSoftware/arlowp-plugin/blob/bbb3ded7ec43e5d2749f33a8c6a54040128f13ba/admin/class-arlo-for-wordpress-admin.php#L176)

**Review by mcdong429 — COMMENTED** [review]
Looks ok to me.

**Review by mcdong429 — APPROVED** [review]

**Pull request merged**